src/device-registry/bin/www

```diff
@@ -147,11 +147,11 @@ const run = async() => {
                         logObject("responseFromInsertMeasurements", responseFromInsertMeasurements)
 
                         if (responseFromInsertMeasurements.success === false) {
-                            logger.error(
-                                `KAFKA: responseFromInsertMeasurements --- ${JSON.stringify(
-                      responseFromInsertMeasurements
-                    )}`
-                            );
+                            //         logger.error(
```

Comment:
I am just commenting out the Slack logging stuff so that our Slack channel is LESS noisy (spammy) @BenjaminSsempala

Reply:
this is the part where the consumed measurements from KAFKA are NOT successfully inserted in device registry. @BenjaminSsempala